client/src/components/MultiJurisdictionAudit/RoundManagement/generateSheets.ts

```diff
@@ -79,12 +79,14 @@ export const downloadPlaceholders = (
     ballots.forEach(ballot => {
       if (pageCount > 0) placeholders.addPage('letter')
 
+      const imprintedId = ballot.imprintedId ? [ballot.imprintedId] : []
       const lines = [
         ballot.auditBoard!.name,
         ballot.batch.container && `Container: ${ballot.batch.container}`,
         ballot.batch.tabulator && `Tabulator: ${ballot.batch.tabulator}`,
         `Batch: ${ballot.batch.name}`,
         `Ballot Number: ${ballot.position}`,
+        ...imprintedId,
```

Comment:
for consistency, i would use the pattern established above on lines 85 and 86 to add this item to the list (falsey values are filtered out on L91)

Reply:
Oh right, somehow I missed that. :stuck_out_tongue_closed_eyes: